### PR TITLE
Add node auto provision for AWS terraform module 

### PR DESCRIPTION
**Pull request by sumanthreddy29**

### CODE REVIEW

File: scenarios/perf-eval/nap-10n100p/scripts/user_data/nodepool.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+apiVersion: karpenter.sh/v1
```
Comment (anson627) [inline scenarios/perf-eval/nap-10n100p/scripts/user_data/nodepool.yml:1]: delete nodepool part

File: scenarios/perf-eval/nap-10n100p/scripts/user_data/deployment.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+apiVersion: apps/v1
```
Comment (anson627) [inline scenarios/perf-eval/nap-10n100p/scripts/user_data/deployment.yml:1]: delete

File: scenarios/perf-eval/nap-10n100p/scripts/user_data/nodepool.yml (hunk 1 above)
Comment (sumanthreddy29) [reply scenarios/perf-eval/nap-10n100p/scripts/user_data/nodepool.yml:1]: removed

File: scenarios/perf-eval/nap-10n100p/scripts/user_data/deployment.yml (hunk 2 above)
Comment (sumanthreddy29) [reply scenarios/perf-eval/nap-10n100p/scripts/user_data/deployment.yml:1]: done

**Pull request closed without merging**